src/main/java/chatbot/tasks/ToDoTask.java

```diff
@@ -0,0 +1,19 @@
+package chatbot.tasks;
+
+public class ToDoTask extends Task {
+    private static String type = "T";
```

Comment:
Same as before, would be good if the naming style reflects the constant nature: `String TYPE`